src/splat/segtypes/segment.py

```diff
@@ -259,6 +259,8 @@ def __init__(
         self.sibling: Optional[Segment] = None
         self.data_sibling: Optional[Segment] = None
         self.rodata_sibling: Optional[Segment] = None
+        self.bss_sibling: Optional[Segment] = None
+        self.siblings: Dict[str, Segment] = {}
```

Comment:
with this new dict, could we get rid of the dedicated "sibling", "data_sibling", "rodata_sibling", "bss_sibling" fields?

Reply:
Removed `data_sibling`, `rodata_sibling` and `bss_sibling`. Plain `sibling` is still useful since it points to the "base segment"